src/TestFramework/DataFaker.php

```diff
@@ -1076,4 +1076,52 @@ public function riskScore()
     {
         return $this->intBetween(-2, 100);
     }
+
+    /**
+     * Return a payment instrument name.
+     *
+     * @return string
+     */
+    public function paymentInstrumentName()
+    {
+        return $this->randomCharacters(self::ALPHABET_UPPER . self::DIGITS, $this->intBetween(4, 10));
+    }
+
+    /**
+     * Return a payment network.
+     *
+     * @return string
+     */
+    public function paymentNetwork()
+    {
+        return $this->randomCharacters(self::ALPHABET_UPPER . self::ALPHABET_LOWER, $this->intBetween(4, 16));
+    }
+
+    /**
+     * Return payment data.
+     *
+     * @return array
+     */
+    public function paymentData()
+    {
+        return array(
+            'version' => $this->randomCharacters(self::ALPHABET_UPPER . self::DIGITS, $this->intBetween(1, 5)),
+            'data' => $this->randomCharacters(self::ALPHABET_UPPER . self::DIGITS, $this->intBetween(4, 120)),
+            'signature' => $this->randomCharacters(self::ALPHABET_UPPER . self::DIGITS, $this->intBetween(4, 20)),
+            'header' => array(
```

Comment:
Travis fails when I use the new array syntax.

Reply:
yeah omnipay supports php 5.2 which does not have new array syntax